extensions/micrometer/deployment/src/test/java/io/quarkus/micrometer/test/Util.java

```diff
@@ -53,4 +54,11 @@ public static String listMeters(MeterRegistry registry, String meterName, final
                 })
                 .collect(Collectors.joining(","));
     }
+
+    public static <T> void waitForMeters(Collection<T> collection, int count) throws InterruptedException {
```

Comment:
Instead of using this, can't we use `Awaitility`? We use it a lot in tests throughout the codebase

Reply:
+1, pretty sure I've used it in some places as well

Reply:
Not going to use awaitility. I don't want it to throw on this condition at all.

It is an "attempt" .. wait at most.. 
Then the next tests are very explicit/clear on what URIs or tags are missing.